google/resource_container_cluster.go

```diff
@@ -514,6 +536,30 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 
 	d.Partial(true)
 
+	if d.HasChange("master_authorized_networks_config") {
+		if c, ok := d.GetOk("master_authorized_networks_config"); ok {
```

Comment:
This means that if a user removes the `master_authorized_networks_config` block from their config, nothing actually changes. Is that intentional? I'd assume it would remove the configuration.

Reply:
That was definitely not intentional, and is now fixed.